sources/kafka/__init__.py

```diff
@@ -82,10 +83,19 @@ def kafka_consumer(
 
     # read messages up to the maximum offsets,
     # not waiting for new messages
+    attempts = 0
     with closing(consumer):
         while tracker.has_unread:
+            if attempts == 30:
```

Comment:
Better to use higher number of attempts, as sometimes, when we just start reading, it takes several seconds before the consumer is directed to a broker.